src/ingestion/models/ingestionManager.ts

```diff
@@ -88,10 +88,14 @@ export class IngestionManager {
         logContext: logCtx,
         metadata: { originDirectory, fileNames, isValid: validationResult.isValid },
       });
+      trace.getActiveSpan()?.setStatus({ code: SpanStatusCode.OK });
```

Comment:
use the `getActiveSpan()` once and reuse it as variable

Reply:
done